### PR TITLE
test(general): Use new kopia upgrade cli in robustness upgrade tests

### PR DESCRIPTION
**Pull request by chaitalisg**
Use `repository upgrade` command in the robustness tests.

### CODE REVIEW

File: tests/robustness/snapmeta/kopia_snapshotter.go [hunk 1]
```diff
@@ -200,7 +201,12 @@ func (ks *KopiaSnapshotter) GetRepositoryStatus() (cli.RepositoryStatus, error)
 // UpgradeRepository upgrades the given kopia repository
 // from current format version to latest stable format version.
 func (ks *KopiaSnapshotter) UpgradeRepository() error {
-	_, _, err := ks.snap.Run("repository", "set-parameters", "--upgrade")
+	os.Setenv("KOPIA_UPGRADE_LOCK_ENABLED", "1")
```
Comment (adowair) [inline tests/robustness/snapmeta/kopia_snapshotter.go:204]: Let's make sure this does not clobber future tests or any other runtime. Is there a way to have a "local env session"?

Comment (adowair) [reply tests/robustness/snapmeta/kopia_snapshotter.go:204]: On way to do this is
``` go
defer os.unsetenv(...)
os.Setenv(...)
ks.snap.run(...)
```

or perhaps to get and set this variable outside this function using `testing.T.Setenv()`. See [this page](https://www.reddit.com/r/golang/comments/ar5z3i/how_to_set_env_variables_while_unit_testing/) for inspiration.

Comment (chaitalisg) [reply tests/robustness/snapmeta/kopia_snapshotter.go]: Added setenv to cleanup function in the main test. Testing.T.Setenv() is not available in main test.

File: tests/robustness/robustness_test/main_test.go [hunk 2]
```diff
@@ -62,6 +62,9 @@ func TestMain(m *testing.M) {
 
 		prev := rs.ContentFormat.MutableParameters.Version
 
+		// This variable is reset in cleanup function.
+		os.Setenv("KOPIA_UPGRADE_LOCK_ENABLED", "1")
+
 		log.Println("Old repository format:", prev)
 		th.snapshotter.UpgradeRepository()
```
Comment (adowair) [inline tests/robustness/robustness_test/main_test.go:69]: Nit: the success or failure of this function is now contingent on doing some work outside it. Is this intended? Could that be confusing in the future, say if someone wants to reuse this function?

File: tests/robustness/robustness_test/main_test.go [hunk 3]
```diff
@@ -32,7 +32,7 @@ const (
 
 var (
 	randomizedTestDur = flag.Duration("rand-test-duration", defaultTestDur, "Set the duration for the randomized test")
-	repoPathPrefix    = flag.String("repo-path-prefix", "", "Point the robustness tests at this path prefix")
+	repoPathPrefix    = flag.String("repo-path-prefix", "/Users/chaitali.gondhalekar/Work/Kasten/kopia_dummy_repo/", "Point the robustness tests at this path prefix")
```
[the remark this answers is not in the document]
Comment (chaitalisg) [reply tests/robustness/robustness_test/main_test.go]: Good catch. Removed it.

File: tests/robustness/robustness_test/main_test.go (hunk 2 above)
Comment (chaitalisg) [reply tests/robustness/robustness_test/main_test.go]: Moved the setEnv call to the method and added the reset to cleanup function.

**Review by adowair — APPROVED** [review]
barring linter errors, looks good to me

**Pull request merged**